tv.html

```diff
@@ -0,0 +1,239 @@
+        <a class="btn btn-white btn-outline tv-btns" href="https://www.amazon.com/Crossroads-Church/dp/B07D83Z1JB"><img
+            src="https://crds-media.imgix.net/3gazTXowti629q6Ai9WW3/a6311faf7f8851b6356e8f598b135443/Logo_Amazongfire.png"></a>
+        <a class="btn btn-white btn-outline tv-btns"
+          href="https://appadvice.com/tv/app/crossroads-anywhere/1029478803"><img class="h-100"
+            src="https://crds-media.imgix.net/5TNLQmLrg05eBQVCEPM8o4/3315e8ce68ac1f9ef75523cb6966a450/Apple_TV_logo.png"></a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="push-top">
+  <div class="container">
+    <div class="col-sm-4 push-bottom">
+      <crds-portrait-card href="https://www.crossroads.net/about/" theme="default" lead="curious about crossroads?"
+        title="Learn About Us"
+        image-src="//crds-media.imgix.net/tM2aWSUtcjYXKcaqY5ZQE/86dcae82f57d935535fde7c5a3cbb20a/what-we-believe-img.png">
+      </crds-portrait-card>
+    </div>
+    <div class="col-sm-4 push-bottom">
+      <crds-portrait-card href="https://www.crossroads.net/locations/" theme="default" lead="Find Us"
+        title="One Church. Many Locations."
+        image-src="//crds-media.imgix.net/A49b0SJWVrG91ooAWaxl0/890de261247941d3057c0ffb6ad97b76/many-locations-img.png">
+      </crds-portrait-card>
+    </div>
+    <div class="col-sm-4 push-bottom">
+      <crds-portrait-card href="https://www.crossroads.net/app/" theme="default" lead="Download the App"
+        title="Connect with God & Pray with Others"
+        image-src="//crds-media.imgix.net/2IgH0VWAw8JOXE16OFTZ2n/bfead14597b8b2959e5124194feeba6e/download-app-img.png">
+      </crds-portrait-card>
+    </div>
+  </div>
+</section>
+
+<section class="bg-orange soft-ends">
+  <div class="container text-center soft-ends push-ends">
+    <div class="row soft-ends">
+      <h1 class="font-family-condensed-extra text-uppercase text-white">We'd Love to Chat With You</h1>
+      <crds-button color="blue" text="Chat with us" data-trigger="chat"></crds-button>
+    </div>
+  </div>
+</section>
+
+<div aria-labelledby="Subscribe email form" class="modal fade" id="btSubscribeModalForm" role="dialog" tabindex="-1">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header hard-bottom"><button aria-label="Close" class="close pull-right" data-dismiss="modal"
+          type="button"><span aria-hidden="true">&times;</span></button>
+      </div>
+      <div class="modal-body text-left hard-bottom">
+        <p class="component-header text-center text-gray-dark push-ends soft-bottom">Get
+          Words from Brian straight to your inbox.</p>
+        <div class="form" data-role="hsform">
+          <!--[if lte IE 8]>
+                          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
+                          <![endif]-->
```

Comment:
Does this need to be here?

Reply:
I think? It is included in every instance you'll find of an embedded Hubspot form, but I don't really know what it does @silasbaur